packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -65,9 +85,19 @@ const Category = props => {
 
     useEffect(() => {
         setTotalPages(totalPagesFromData);
+        return () => {
+            setTotalPages(null);
+        };
     }, [totalPagesFromData]);
 
-    if (error) return <div>Data Fetch Error</div>;
+    // Retry 3 times and then display the fallback error.
```

Comment:
3 is an arbitrary retry count. If ya'll want I can move it to a constant and change it. 🤷‍♂

Reply:
Well, my rule is, "if it shows up three times, abstract it". For this type of slightly nonintuitive logic, I think maybe we abstract it after the _second_ time it crops up. So this is fine for now.